docs/modules/general/proto.mdx

```diff
@@ -0,0 +1,286 @@
+        </td>
+        <td>Filters are aware of who owns TNT.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.4</label>
+        </td>
+        <td>Wools must have a location property.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.3</label>
+        </td>
+        <td>Define how overlapping regions should behave.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.2</label>
+        </td>
+        <td>Added monument modes.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.1</label>
+        </td>
+        <td>Off-by-one region bug fixed.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Map Protocol Changelog
+
+### Changes in 1.4.2
+
+<span className="badge badge--danger">Breaking</span>
+<br />
+
+- The objectives filter will now always automatically derive team ownership for the objective from context. To match an objective without any specific team, the <label>any="true"</label> attribute needs to be specified.
+  ***
+
+<span class="badge badge--success">New</span>
+<br />
+
+<span className="badge badge--secondary">Partially implemented</span>{" "}
+
+- Portals, kits, score boxes & structures can now be dynamically applied using dynamic filters. All region types are dynamic, however, only some filter types are dynamic.
+  - Kits now have [`give`, `take` & `lend` properties](/docs/modules/gear/kits#dynamic-kits).
+  - Portals have [`forward`, `reverse` & `transit` properties](/docs/modules/mechanics/portals#portal-sub-elements).
+  - Structures & [score boxes](/docs/modules/objectives/scoring#score-box-attributes) have a `trigger` property.
+- <span className="badge badge--secondary">Not implemented</span> Added a player
+  count filter.
+- Added a `<grounded/>` filter to check if the player is standing on the ground.
+- Added `<match-started/>`, `<match-running/>` & `<match-finished/>` filters that are used to check the state of the current match.
+- New `pre-match-physics` attribute for the terrain module to enable physics events before the match starts.
+- Added `<observing>` and `<participant>` filters. Also added an `observers` filter property to portals to restrict observer access.
+- Added player rank and score filters. Filters return if the player's rank or score is within the specified range.
+- Control points have new `recovery` and `decay` attributes that replace the `incremental` attribute and allow more control of the progress.
+
+### Changes in 1.4.0
+
+<span className="badge badge--danger">Breaking</span>
+<br />
+
+- Filters, regions, and kits now use `id` instead of `name`.
+  - Keep in mind that IDs are all in the same namespace, so you can not use the same ID for two different types of thing.
+- Teams have an `id` attribute and are always referenced by it everywhere in the XML, **never** by name.
+- Standalone filter definitions are no longer wrapped in a `<filter>` tag, they start with an actual filter, just like regions, e.g.
+
+  - ```xml
```

Comment:
Move all of these XML examples into the same relevant top bullet. Do not give it a separate bullet.